CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+Unreleased
+==========
+
+* Implement option to get notifications also for messages not just for mentions
```

Comment:
@nickel715 if you're already making a commit for something else, then you could. It's not a big concern.